code-rules.tex

```diff
@@ -58,42 +58,33 @@ \section*{Notes}
 \section*{Abstract}
 
 Loss of key personnel has always been a risk for research software projects.
-Key members of the team may have to step away due to illness or burnout,
-to care for a family member,
-from a loss of financial support,
-or because their career is going in a new direction.
-Today,
-though,
-political and financial changes are putting large numbers of researchers out of work simultaneously,
-potentially leaving large amounts of research software abandoned.
-In this article,
-we present ten tips to help researchers ensure that
-the software they have built will continue to be usable
-after they have left their present job.
+Key members of the team may have to step away due to illness or burnout, to care for a family member, from a loss of financial support, or because their career is going in a new direction.
+Today, though, political and financial changes are putting large numbers of researchers out of work simultaneously, potentially leaving large amounts of research software abandoned.
+In this article, we present ten tips to help researchers ensure that the software they have built will continue to be usable after they have left their present job.
 
 \linenumbers
 
 \section*{Introduction}
 
 Academic life is often precarious,
-not least for those who work on research software,
+particularly for those who work on research software,
 an asset that is essential to modern research \cite{Pearson2025}
 but often underfunded, undercited, and overlooked \cite{Carver2022}.
-The situation has recently worsened dramatically due to changes in the political and economic climate:
```

Comment:
This appears to be undoing the change from #56 here.

Reply:
Thanks. Realized after submitting I could've done this better and in smaller chunks. I'll fix this one.